model/lesson.js

```diff
@@ -170,4 +221,5 @@ module.exports = {
     createLesson: createLesson,
```

Comment:
Moving forward, we don't need to write the name of the function twice, just write it once i.e. { createLesson } instead of { createLesson: createLesson }